tests/unit/elements__header.js

```diff
@@ -1,30 +1,50 @@
-define('tests/unit/elements--header',
-    ['elements/header', 'jquery', 'tests/unit/helpers'],
-    function(Header, $, helpers) {
+/*
+   Module named with underscores since Karma can't find it when it has hyphens
+   for some reason.
+*/
+define('tests/unit/elements__header',
```

Comment:
I tried making this `--` locally and the file gets loaded but the contents of the `define` is never evaluated. I don't understand what's going on there...